docs/fsharp/language-reference/literals.md

```diff
@@ -39,6 +39,11 @@ The following table shows the literal types in F#. Characters that represent dig
 
 Values that are intended to be constants can be marked with the [Literal](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-core-literalattribute.html) attribute. This attribute has the effect of causing a value to be compiled as a constant.
 
+Named literals are useful for 
+- pattern matching without a `when` clause
+- attribute arguments
+- static type provider arguments
```

Comment:
```suggestion
Named literals are useful for:
- Pattern matching without a `when` clause.
- Attribute arguments.
- Static type provider arguments.
```

Or maybe like this?

Named literals are useful for pattern matching without a `when` clause, attribute arguments, and static type provider arguments.

Reply:
The version that matches the rest of the style of the documentation best.